upgrade-1.x-to-2.x.sh

```diff
@@ -76,6 +90,14 @@ else
     esac
 fi
 
+# Find boot path
+boot_path=$(findmnt -n --raw --evaluate --output=target /dev/${BOOT_PARTITION})
```

Comment:
Can we find by label here? e.g.
findmnt -n --raw --evaluate --output=target LABEL=resin-boot